ui/src/views/Scopes/ScopesetComparison/index.jsx

```diff
@@ -99,7 +124,7 @@ export default class ScopesetComparison extends Component {
     const { scopeTextA, scopeTextB, scopesetDiff, cellColors } = this.state;
 
     return (
-      <Dashboard title="Compare Scopes">
+      <Dashboard title="Compare Scopesets">
```

Comment:
I think we can keep whatever the latest master has this value as.

Reply:
I don't ever recall changing this title, but I have reverted the change.